src/components/create-vm-dialog/uiState.js

```diff
@@ -31,6 +31,18 @@ const CREATE_TIMEOUT = 'CREATE_TIMEOUT';
 
 const timeouts = { session: {}, system: {} };
 
+export function setVmCloneInProgress(name, connectionName) {
+    const vm = {
+        name,
+        connectionName,
+        isUi: true,
+        cloneInProgress: true,
+    };
+
+    store.dispatch(addUiVm(vm));
+    setupCleanupTimeout(name, connectionName, CREATE_TIMEOUT);
```

Comment:
Is that really desirable? Creating a new VM is somewhat predictable, but cloning a VM might take arbitrarily long if there is lots of data to copy?